src/main.js

```diff
@@ -12,6 +12,7 @@ import { faArrowLeft, faArrowRight } from '@fortawesome/free-solid-svg-icons';
 // social icons
 import { faSquareXTwitter, faFacebookSquare, faGithub, faFlickr, faYoutubeSquare, faInstagram } from "@fortawesome/free-brands-svg-icons";
 library.add(faSquareXTwitter, faFacebookSquare, faGithub, faFlickr, faYoutubeSquare, faInstagram);
+library.add(faArrowRight, faArrowLeft)
```

Comment:
Where are these used?

Reply:
These are the button navigation arrows at the bottom of the page

Reply:
ohhhh.... they just aren't showing on the current live site. Got it

Reply:
they work in the site now! 👍